runner/runners/bazel.go

```diff
@@ -150,6 +150,25 @@ func setupJDKSymlink(path, filename string) error {
 	return os.Symlink(jh, filepath.Join(path, filename))
 }
 
+// API indicates directories where output files and directories would be created exist before execution
+func createOutputPaths(cmd *runner.Command, coDir string) error {
```

Comment:
this is already in master too... is something funky going on with this PR?

Reply:
Hmm, weird - when I do git diff master on my branch it doesn't show changes. I did a rebase after the previous PR and it asked me to merge my branch with my branch...

Reply:
Yeah, in the commit list (here and on my branch) the commits got doubled because of the merge. I think squash and merge should resolve that, though.